dist/default-build-script/Assets/Editor/UnityBuilderAction/Versioning/VersionApplicator.cs

```diff
@@ -15,7 +15,12 @@ public static void SetVersion(string version)
     }
 
     public static void SetAndroidVersionCode(string androidVersionCode) {
-      PlayerSettings.Android.bundleVersionCode = Int32.Parse(androidVersionCode);
+      int bundleVersionCode = Int32.Parse(androidVersionCode);
+      if (bundleVersionCode <= 0) {
+        return;
+      }
```

Comment:
I can't think of any case, but are you sure this won't break anyone's workflow?

Reply:
Yes, the [version code must be a positive integer >=0](https://developer.android.com/studio/publish/versioning) 

If the user is building for the very first time, they should start with `androidVersionCode = 1`, so if they try to set `androidVersionCode = 0`, then the build will use Unity's default for the version code (ie the value inside their `ProjectSettings` folder).